### PR TITLE
Feature/light dark mode

### PR DESCRIPTION
**Pull request by JamesDrysdale**
This PR implements the following features:

- Light and Dark Mode toggles between relevant themes
- The download CV button can be clicked to download the CV
- The mobile menu toggle now shows/hides the mobile menu, and it will close either when an option is selected, or the cross is clicked

Previews:

Screenshot 2023-09-07 at 21 01 23
Screenshot 2023-09-07 at 21 01 31
Screenshot 2023-09-07 at 21 02 28
Screenshot 2023-09-07 at 21 02 41
Screenshot 2023-09-07 at 21 04 33
Screenshot 2023-09-07 at 21 04 44

### CODE REVIEW

File: components/Navbar.tsx [hunk 1]
```diff
@@ -38,32 +55,57 @@ const Navbar = () => {
...
+            alt='light and dark mode toggle'
             className='cursor-pointer'
-            onClick={() => {}}
+            onClick={() => setTheme(currentTheme === 'dark' ? 'light' : 'dark')}
           />
         </div>
       </section>
+      {showMobileNavbar && (
+        <div
+          className='fixed z-40 flex h-screen w-screen justify-center bg-black/30'
```
Comment (cdsalty) [inline components/Navbar.tsx:97]: Did you mean to use `bg-black300` instead of `bg-black/30`?

**Review by cdsalty — APPROVED** [review]
This is looking good @JamesDrysdale. I came across one possible typo, but other than that, really great job!

File: components/Navbar.tsx (hunk 1 above)
Comment (snlaight) [reply components/Navbar.tsx:97]: Hey @cdsalty just chiming in here , but setting a / property in a color in tailwind is specifying an opacity layer . So by saying bg-black/30 , it's a shorthand property to say set the background color to black with 30% opacity

Comment (JamesDrysdale) [reply components/Navbar.tsx:97]: Yes, I did, great spot. :)

**Pull request merged**